### PR TITLE
Fixed bugs 

### PR DESCRIPTION
**Pull request by vyskocilpavel**
* Added CHANGELOG.md
* Added composer.json file
* Fix the identifier name in dictiona
* Removed twig templatery
* Renamed module to remoteUserSSL
* Fixed namesmaces to use with simpleSAMLphp in version < 1.17.0

### CODE REVIEW

File: lib/Auth/Source/RemoteUserSSL.php [hunk 1]
```diff
@@ -61,20 +61,19 @@ public function authenticate(&$state) {
...
+            \SimpleSAML\Logger::warning('remoteUserSSL: no matching user found in LDAP for login='.$login);
             $this->authFailed($state);
 
             assert(false); // should never be reached
             return;
         }
 
         \SimpleSAML\Logger::info('authRemoteUserSSL: '.$dn);
-	$attributes = $this->ldapcf->getAttributes($dn);
+	    $attributes = $this->ldapcf->getAttributes($dn);
```
Comment (tauceti2) [inline lib/Auth/Source/RemoteUserSSL.php]: unnecessary indentation

File: lib/Auth/Source/RemoteUserSSL.php [hunk 2]
```diff
@@ -61,20 +61,19 @@ public function authenticate(&$state) {
...
         assert(is_array($attributes));
         $state['Attributes'] = $attributes;
 
-	$this->authSuccesful($state);
+	    $this->authSuccesful($state);
```
Comment (tauceti2) [inline lib/Auth/Source/RemoteUserSSL.php]: unnecessary indentation

File: lib/Auth/Source/RemoteUserSSL.php [hunk 3]
```diff
@@ -102,13 +101,18 @@ public function authSuccesful(&$state) {
...
-        $t = new \SimpleSAML\XHTML\Template($config, 'authRemoteUserSSL:error.php');
+        $t = new SimpleSAML_XHTML_Template($config, 'remoteUserSSL:RemoteUserSSLerror.php');
         $t->data['loginurl'] = \SimpleSAML\Utils\HTTP::getSelfURL();
-        $t->data['errorcode'] = $state['authRemoteUserSSL.error'];
+        if (isset($state['remoteUserSSL.error'])) {
+            $t->data['errorcode'] = $state['remoteUserSSL.error'];
+        }
         $t->data['errorcodes'] = \SimpleSAML\Error\ErrorCodes::getAllErrorCodeMessages();
 
+        $t->data['spEntityId'] =$state['core:SP'];
```
Comment (tauceti2) [inline lib/Auth/Source/RemoteUserSSL.php]: missing space after equals

File: lib/Auth/Source/RemoteUserSSL.php (hunk 1 above)
Comment (vyskocilpavel) [reply lib/Auth/Source/RemoteUserSSL.php]: Solved

File: lib/Auth/Source/RemoteUserSSL.php (hunk 3 above)
Comment (vyskocilpavel) [reply lib/Auth/Source/RemoteUserSSL.php]: Resolved

File: lib/Auth/Source/RemoteUserSSL.php (hunk 2 above)
Comment (vyskocilpavel) [reply lib/Auth/Source/RemoteUserSSL.php]: Resolved

**Pull request merged**